### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ujwalpawar913**

### CODE REVIEW

**Review by lohitpenu — APPROVED** [review]

File: Codes/coordinate_initialization.py [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+#  * Output: none 
+#  * Logic: will take photo and process it 
+#			to look for red border.
+#			if found will call go_for_y
+#			after changing dist_y_bord
+#			as calculated
+#  * Example Call: image_process_for_coord()
+#****************************************
+
+def image_process_for_coord():
```
Comment (k3d61) [inline Codes/coordinate_initialization.py:166]: can you please add the explanation of your algorithm?
1. add explnation here, or
2. add it in documentation and mention that here.
In explanation, write about purpose of every function that you are writing as you have already for most of the functions.

File: Codes/coordinate_initialization.py [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+global distance
+global found_red
+global found_black
+global mm_per_pix
+global coord_ini_status
+
+#************Initialization*************
+
+#ser = serial.Serial('/dev/ttyACM0',9600)
+phase=1
```
Comment (k3d61) [inline Codes/coordinate_initialization.py]: explain what is this phase variables's porpose

File: Codes/coordinate_initialization.py [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+	ret, img = cap.read()
+	if cap.isOpened() :
+		hsv=cv2.cvtColor(img,cv2.COLOR_BGR2HSV)
+		if phase == 1 :
+			lower_range=np.array([160,50,50])
+			higher_range=np.array([180,255,255])
+		elif phase == 2:
+			lower_range=np.array([0,50,50])
+			higher_range=np.array([0,255,255])
+		mask1 = cv2.inRange(hsv,lower_range,higher_range)
```
Comment (k3d61) [inline Codes/coordinate_initialization.py:176]: please change the name of mask1 --> to something meaningful

File: Codes/coordinate_initialization.py [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+# 			and communicate to arduino, followed
+#			by confirmation  (for y direction)
+#  * Example Call:go_for_y()
+#****************************************
+def go_for_y(final):
+	# communication string : @ + x_coord + y_coord + z_coord + seed_bit + water_bit + initialization_bit + !
+	arduino_confirm=False
+	mystring=""
+	if final == 0 :
+		comm_string="@"+"0000"+str(dist_y_bord).zfill(3)+"000"+"002"+"!"
```
Comment (k3d61) [inline Codes/coordinate_initialization.py:117]: this code seems to be reused in go_for_x you can make seperate function out of it

**Pull request merged**